src/style.css

```
@@ -0,0 +1,119 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  --color-button: #00acdc;
+  --color-white: #fff;
+  --color-border: #f3f3f3;
+  --color-text-black: #060b35;
+  --color-text-grey: #616070;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  margin: 28px;
+  width: 200px;
+}
+
+.card {
+  background: var(--color-white);
+  border: 1px solid var(--color-border);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-black);
+}
+
+.card__code {
+  padding: 4px 0 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-grey);
+}
+
+.card__stars-reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars .stars--star,
+.stars .stars__star:nth-child(-n+4),
+.stars .stars__star:nth-child(-n+3),
+.stars .stars__star:nth-child(-n+2),
+.stars .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-black);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 24px;
+  padding-bottom: 16px;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-grey);
+}
+
+.card__price-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-text-black);
+}
+
+.card__button-link {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 38px;
+  height: 38px;
```

```suggestion
  height: 40px;
```
add box-sizing: border-box on * selector